## 50% of end-end tests have passed
```
Total executed tests: 26
Failed tests: 13
Expected to fail tests: 0
Passed tests: 13
```
```
IDs of failed tests: 727230,702786,703629,702948,702809,703194,702958,703503,702957,702845,702775,704613,727229 
```
<h3>Failed tests (13)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/android-permission&PR_ID=20531&APK_NAME=20531.apk&TR_CASE_IDS=727230,702786,703629,702948,702809,703194,702958,703503,702957,702845,702775,704613,727229">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584786">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 2: Looking for a message by text: gallery </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:454: in test_community_several_images_send_reply
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Gallery message was not received
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#236">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584768">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the deep link status.app://c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/05212e9a97324a90807e53ce6228a2be?auth=b8c6d11256e5666e1383f4b33be7cbe3#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584789">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `accessibility id`: `community-title` </div><div>Device 1: `Text` is `Closed community` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the url https://status.app/c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/05212e9a97324a90807e53ce6228a2be?auth=b8c6d11256e5666e1383f4b33be7cbe3#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584792">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:151: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584791">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:131: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584787">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community tags in the Discover communities screen</div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'Status' is not in the Discover Communities list.
E    Community 'Status Inu' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/21bba3fcf4b648d3aaa77c74bdae738b?auth=89180706b4d25c05380202f036053f8d#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584770">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/07e94e01b57d4ea3a01fd4724e04c84c?auth=36b29a784606efcd22a0e0b445cb5d5d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f3a8aec985114b92bd7c79d54614372c?auth=ff79460c91feab1e32df2b441e701e6a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584788">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584782">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584771">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584778">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584784">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584783">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e200f74f4fbf4bf29f2186e5b7395427?auth=62c7226e6d67593e7907b93d33883059">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6b602151eb384be5983bafd65b1bcb4f?auth=09da564512addda0672817ee7f2194ca">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (13)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584769">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ede8eb7dbf4b40729eef42340512f536?auth=d98be1b4a6f833f652dddf79f2b621f5#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e665d8c3235a449c878414ea3a501e72?auth=9b18c079f71f69779c55e6ccc435ad79#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584767">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ede8eb7dbf4b40729eef42340512f536?auth=d98be1b4a6f833f652dddf79f2b621f5#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e665d8c3235a449c878414ea3a501e72?auth=9b18c079f71f69779c55e6ccc435ad79#130">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584785">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#551">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#459">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584772">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#163">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584779">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#257">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#211">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584776">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#181">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584777">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#329">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#288">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584773">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#205">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584780">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#424">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#379">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584775">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#394">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#349">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584790">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#586">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#485">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584774">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/01f0b10e807046f6b79da64f51f2a9f9?auth=17fe7874cadd3888cdc57000284bb7ff#305">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dbbf82f855074ddfb7483b8d5a5af172?auth=5c987737e21bf1df20ec3a0fe43d631a#271">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584781">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c3a79262bf5143b2bc9e4e7c6df76a56?auth=cc58fc286ec40c1f796d92138a6d946e#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2ff3c04794d423092d5916ef8c31683?auth=d1fb815cbc081b72e48ac6870c0e0347#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/legacy/status_im/ui/screens/notifications_settings/views.cljs
+++ b/src/legacy/status_im/ui/screens/notifications_settings/views.cljs
@@ -6,6 +6,7 @@
     [legacy.status-im.ui.components.react :as react]
     [quo.core :as quo]
     [react-native.platform :as platform]
+    [taoensso.timbre :as log]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
@@ -57,7 +58,15 @@
        :accessibility-label :local-notifications-settings-button
        :subtitle            (i18n/label :t/local-notifications-subtitle)
        :active              notifications-enabled?
-       :on-press            #(rf/dispatch [:push-notifications/switch (not notifications-enabled?)])
+       :on-press            (fn []
+                              (when-not notifications-enabled?
+                                (rf/dispatch
+                                 [:request-permissions
+                                  {:permissions [:post-notifications]
+                                   :on-allowed  #(log/info "push notification permissions were allowed")
+                                   :on-denied   #(log/error
+                                                  "user denied push notification permissions")}]))
+                              (rf/dispatch [:push-notifications/switch (not notifications-enabled?)]))
        :accessory           :switch}]]))
 
 (defn notifications-settings

--- a/src/status_im/contexts/onboarding/enable_notifications/view.cljs
+++ b/src/status_im/contexts/onboarding/enable_notifications/view.cljs
@@ -2,7 +2,6 @@
   (:require
     [quo.core :as quo]
     [react-native.core :as rn]
-    [react-native.platform :as platform]
     [react-native.safe-area :as safe-area]
     [status-im.common.resources :as resources]
     [status-im.contexts.onboarding.enable-notifications.style :as style]
@@ -36,7 +35,7 @@
                                 {:permissions [:post-notifications]
                                  :on-allowed  #(log/info "push notification permissions were allowed")
                                  :on-denied   #(log/error "user denied push notification permissions")}])
-                              (rf/dispatch [:push-notifications/switch true platform/ios?])
+                              (rf/dispatch [:push-notifications/switch true])
                               (rf/dispatch [:navigate-to-within-stack
                                             [:screen/onboarding.welcome
                                              :screen/onboarding.enable-notifications]]))

